### PR TITLE
[AC-411] Explain login process

### PR DESCRIPTION
**Pull request by 0581raghav**

### CODE REVIEW

File: features.md [hunk 1]
```diff
@@ -5,6 +5,18 @@
...
+   1. Entering the server URL as  https://demo.openmrs.org/openmrs 
+   ![OpenMRS Android Client Offline login screen](https://user-images.githubusercontent.com/24738188/34084262-1732ca6c-e3a3-11e7-9e05-96d1ce0835c3.png) 
+   
+   2. Now enter the username as admin and password as Admin123 .
+   ![OpenMRS Android Client Offline login screen](https://user-images.githubusercontent.com/24738188/34084270-23c24654-e3a3-11e7-8641-619a71d85739.png)
+   
+   3. Select the Session location and also choose the Online/Offline option.
+   
+   4. Now click on Login. Now you will be redirected to the home page.
+   ![OpenMRS Android Client Offline login screen](https://user-images.githubusercontent.com/24738188/34084273-28d474aa-e3a3-11e7-90f1-218b29e1d36f.png)
```
Comment (shivtej1505) [inline features.md]: Could you please make images somewhat smaller? They are taking too much space.

File: features.md [hunk 2]
```diff
@@ -5,6 +5,18 @@
    <!-- Explain how data is stored and secured on the device and the SSL connection with the server -->
    
 * ### Login
+   1. Entering the server URL as  https://demo.openmrs.org/openmrs 
+   ![OpenMRS Android Client Offline login screen](https://user-images.githubusercontent.com/24738188/34084262-1732ca6c-e3a3-11e7-9e05-96d1ce0835c3.png)
```
Comment (shivtej1505) [inline features.md]: Please change the URL shown in the screenshot to `https://demo.openmrs.org/openmrs` otherwise, it will create confusion.

**Review by shivtej1505 — CHANGES_REQUESTED** [review]
Nice progress :)
Please squash the two commits into one.

**Comment by shivtej1505** [comment]
@0581raghav, can you squash the four commits into one?

**Comment by dkayiwa** [comment]
@0581raghav did you see the merge conflicts?

**Comment by 0581raghav** [comment]
@dkayiwa  Sorry, I will update the pull request after resolving merge conflict.

**Comment by f4ww4z** [comment]
@0581raghav Just saw your fork is behind this repo's master. You can [see here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to pull the latest changes first :+1: .

**Review by shivtej1505 — APPROVED** [review]

**Comment by shivtej1505** [comment]
Thanks, @0581raghav, merging now!

**Pull request merged**